client/me/purchases/manage-purchase/style.scss

```diff
@@ -55,6 +61,14 @@
 		.manage-purchase__title {
 			width: 60%;
 		}
+
+		@include breakpoint( ">660px" ) {
+			.manage-purchase__detail,
+			.manage-purchase__detail-label {
+				height: 40px;
+				width: auto;
```

Comment:
Sorry my comment before wasn't very clear. I'd leave the `width: 25%;` declarations inside a `<660px` breakpoint, but move the height ones to the `660px` breakpoint. This is to avoid unnecessary declarations like `width: auto`. It's not a big deal, it's just about trying to get a balance between keeping the output small, keeping the code clean, and declaring mobile styles first and overriding in desktop.